index.html

```diff
@@ -184,78 +193,76 @@ <h2 class="title-medium">Contact</h2>
                 <div class="row">
                     <div class="col col-6">
                       <label>

```

Comment:
Why do you need `label` as a wrapper for input? I guess it is useless without text description.